website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -0,0 +1,163 @@
+| Only persist audit entries that attempt to perform `read` operations                   | `operation == read`                     |
+| Only persist audit entries for requests that interact with `kv` within namespace `ns2` | `namespace == ns2 and mount_type == kv` |
+
+### Scenario: `kv` only
+
+A Vault Operator wants only `kv` type audit entries to be written to an audit log for a particular device.
+
+1. Enable an audit device with the following filter option:
+
+    ```
+    vault audit enable -path kv-only file file_path=/var/audit.log filter="mount_type == kv"
+    ```
+
+2. List the enabled audit devices:
+
+    ```
+    vault audit list --detailed
+    ```
+
+3. Enable a `kv` secrets engine:
+
+    ```
+    vault secrets enable -path my-kv kv-v2
+    ```
+
+4. Write secret data to the `kv` engine:
+
+    ```
+    vault kv put my-kv foo=bar
+    ```
+
+5. Disable the audit device:
+
+    ```
+    vault audit disable -path kv-only
+    ```
+
+The steps above performed a number of actions against Vault that generate audit entries.
+However, only two of them had a `mount_type` of `kv` (steps 3 and 4).
+
+Viewing the contents of the file audit log at `/var/audit.log` will show that only audit
+entries related to steps 3 and 4, with a request and response audit entry for each.
+
+### Test message
+
+When enabling an audit device, Vault will (by default) attempt to send a test message
+to the device. In cases where filtering is configured on a device, it may be possible
+for the evaluation of the filter to result in the test message failing to meet the specified
+predicate expression, and therefore not being written to the audit device's sink. This test
+message will always be present in `fallback` devices (described below).
+
+The properties of the test message are as follows:
+
+* `mount_point` - not present
+* `mount_type` - not present.
+* `namespace` - not present.
+* `operation` - `update`.
+* `path` - `sys/audit/test`.
+
+Vault Operators should use this when trying to understand when they should, or
+should not expect a test message to appear in the sink for a newly enabled audit device.
+
+## Fallback device
+
+The ability to filter audit entries can provide great flexibility to your workflows,
+however the additional complexity can make it possible for a Vault Operator to configure their
+audit devices in such a way that some audit entries are missed out from audit logs entirely.
+
+The `fallback` audit device is the (non-mandatory) mechanism by which Vault can
+continue to adhere to Vault's [security model](/vault/docs/internals/security),
+that all requests and responses are successfully logged before the client receives
+any secret material.
+
+When exclusively using filtered audit devices, enabling the fallback audit device
+results in any audit entry which would have been ignored/missed due to filter settings
```

Comment:
This has a lot of passive voice so it's hard to parse. I suggest making it more active -- When exclusively using filtered audit devices, enable the fallback audit device to catch any events that would otherwise be missed.